libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -129,7 +132,7 @@ const AP_Param::GroupInfo AP_Follow::var_info[] = {
     // @Param: _ALT_TYPE
     // @DisplayName: Follow altitude type
     // @Description: Follow altitude type
-    // @Values: 0:absolute, 1:relative
+    // @Values: 0:absolute, 1:relative, 2:origin (not used), 3:terrain (plane)
```

Comment:
Why add a unused option?

Reply:
So that the numbers match Location:AltFrame

Reply:
Sure, but you don't need to call out 2 in the description. That will populate a drop down that will allow people to select it.